src/java.base/share/classes/java/util/zip/InflaterInputStream.java

```diff
@@ -251,7 +251,12 @@ public void close() throws IOException {
 
     /**
      * Fills input buffer with more data to decompress.
+     * This method will read up to {@link #buf}.length bytes into the input
+     * buffer, {@link #buf}, starting at element {@code 0}. The {@link #len}
+     * field will be set to the number of bytes read.
```

Comment:
Thanks for the update, this wording looks okay, I'm just wondering if we should add implSpec as "This method will read ..." is really "Implementation Specification".

Reply:
> Thanks for the update, this wording looks okay, I'm just wondering if we should add implSpec as "This method will read ..." is really "Implementation Specification".

Sure thing added the tag